core/Azure.Mcp.Core/src/Areas/Server/Options/ServiceOptionDefinitions.cs

```diff
@@ -41,6 +42,17 @@ public static class ServiceOptionDefinitions
         DefaultValueFactory = _ => (string?)ModeTypes.NamespaceProxy
     };
 
+    public static readonly Option<string[]?> Tool = new Option<string[]?>(
+        $"--{ToolName}"
+    )
+    {
+        Description = "Filter to expose only specific tools by name (e.g., 'azmcp storage account list'). Can be specified multiple times.",
```

Comment:
Add an example of how to list multiple tools and improve overall clarity

```suggestion
        Description = "Expose only specific tools by name (e.g., 'azmcp storage account list'). Repeat this option to include multiple tools, e.g., --tool \"azmcp storage account list\" --tool \"azmcp network vnet list\".",
```